test/built-ins/ShadowRealm/prototype/importValue/throws-typeerror-import-syntax-error.js

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2021 Chengzhong Wu. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-realmimportvalue
+description: >
+  ShadowRealm.prototype.importValue rejects with TypeError when the imported script unable to be parsed.
+info: |
+  RealmImportValue ( specifierString, exportNameString, callerRealm, evalRealm, evalContext )
+
+    ...
+    17. Return ! PerformPromiseThen(innerCapability.[[Promise]], onFulfilled, callerRealm.[[Intrinsics]].[[%ThrowTypeError%]], promiseCapability).
+
+flags: [async, module]
```

Comment:
```suggestion
flags: [async]
```

Reply:
@legendecas the reason for this change is that this must be tested in both non-strict mode and strict mode. The `module` flag will contradict that.

Reply:
I was following the flags here https://github.com/tc39/test262/blob/main/test/built-ins/ShadowRealm/prototype/importValue/throws-if-import-value-does-not-exist.js#L37. Do you agree we should remove it too? I'd believe the condition is similar for throws-if-import-value-does-not-exist.js.

Reply:
I'm really sorry, but I've realized that @leobalter and I were wrong: `module` is currently required for the `test262-harness` + `eshost` infra to resolve any kind of imported code. I will work on trying to update that, but for now we'll have to continue to include the `module` flag (which does follow the rules in INTERPRETING.md)

Reply:
I can fix the other PR that was already landed, unless you get to it before me.

Reply:
@rwaldron updated!

Reply:
@legendecas again, I apologize for the confusion and thank you again for writing these and for doing the other follow up!